tools/src/main/python/dlpx/virtualization/_internal/delphix_client.py

```diff
@@ -20,10 +22,14 @@ class DelphixClient(object):
     """
     __BOUNDARY = '----------boundary------'
     __UPLOAD_CONTENT = 'multipart/form-data; boundary={}'.format(__BOUNDARY)
+    __WAIT_TIME_SECONDS = 5
```

Comment:
nit: This is the polling interval, right? May be change the name to indicate so - JOB_POLLING_INTERVAL?